doc/src/manual/methods.md

```diff
@@ -335,7 +336,20 @@ Here's an example where the method type parameter `T` is used as the type parame
 type `Vector{T}` in the method signature:
 
 ```jldoctest
-julia> myappend(v::Vector{T}, x::T) where {T} = [v..., x]
+julia> function myappend(v::Vector{T}, x::T) where {T}
+           return [v..., x]
+       end
+myappend (generic function with 1 method)
+```
+
+The type parameter `T` in the function's signature ensures that the type of the new `v` is preserved after element `x` is appended to it. 
+The function's signature is preceded by the keyword `where` and is placed immediately after the method arguments. 
+When only one type parameter is used, the curly braces are optional. 
+Parametric method definition also works with the short-form syntax.
+An equivalent one-line definition would be:
```

Comment:
The one-line form is already used at the beginning of the section, so it doesn’t need to be introduced here again. Maybe all of the explanations in this PR should instead be added to the first example in the section (`same_type`) instead of this one.

Reply:
Frankly I think it will be easier if you move the paragraph to the better location. I have now given you write access if you want to use it, please do. I think you understand the parts where I was confused better than me!